README.md

```diff
@@ -78,6 +78,7 @@ your_profile_name:
       use_lw_deletes: [False] Use the strategy `delete+insert` as the default incremental strategy.
       check_exchange: [True] # Validate that clickhouse support the atomic EXCHANGE TABLES command.  (Not needed for most ClickHouse versions)
       local_suffix [_local] # Table suffix of local tables on shards for distributed materializations.
+      local_db_prefix [local_] # Database prefix of local tables on shards for distributed materializations. If empty, it uses the same database as the distributed table.
```

Comment:
I believe the default value should be [] here

Reply:
Well spotted, thanks!